esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6123,6 +6123,12 @@ void CWsDfuEx::getFileAccess(IEspContext &context, IUserDescriptor *udesc, SecAc
     accessInfo.setExpiryTime(expiryTime);
     accessInfo.setFileAccessPort(port);
     accessInfo.setFileAccessSSL(secure);
+
+    StringBuffer userName;
+    if (udesc)
+        udesc->getUserName(userName);
+    LOG(daliAuditLogCat,",FileAccess,,READ,,%s,%s,%s,%s,%d", userName.str(), fileName.str(),
+        req.getCluster(), req.getJobId(), req.getExpirySeconds());
```

Comment:
Example from an existing audit log for a READ:
```
,FileAccess,HThor,READ,hthor,regress,regress::single::c.dg_fetchfilepreloadindexed,W20190125-160407,graph1
```
HThor is the component type. So should be "EspProcess" here.
'hthor' is the cluster name. Should be req.getCluster() 

The fields after the filename are normally graph name, timing, etc.
You don't have those and can't add them, but we need some way to tell what fields being added are going forward.
I suggest adding a field name as a prefix, e.g. ",jobid=blah,expirySecs=10"